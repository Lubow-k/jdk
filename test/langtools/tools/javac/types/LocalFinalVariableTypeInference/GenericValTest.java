/*
 * @test
 * @compile/fail/ref=GenericValTest.out -XDrawDiagnostics GenericValTest.java
 */


import java.util.List;

public class GenericValTest {

    void test() {
        List<val> v1;           // error
        List<? extends val> v2; // error
        List<? super val> v3;   // error
    }
}

