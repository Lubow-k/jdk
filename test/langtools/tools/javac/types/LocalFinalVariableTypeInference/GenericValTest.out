GenericValTest.java:12:14: compiler.err.restricted.type.not.allowed.here: val
GenericValTest.java:13:24: compiler.err.restricted.type.not.allowed.here: val
GenericValTest.java:14:22: compiler.err.restricted.type.not.allowed.here: val
3 errors